dtfs-central-api/src/repositories/tfm-teams-repo/tfm-teams.repo.ts

```diff
@@ -0,0 +1,19 @@
+import { WithoutId, Collection } from 'mongodb';
+import { TeamId, TfmTeam } from '@ukef/dtfs2-common';
+import { mongoDbClient } from '../../drivers/db-client';
+
+export class TfmTeamsRepo {
+  private static async getCollection(): Promise<Collection<WithoutId<TfmTeam>>> {
+    return await mongoDbClient.getCollection('tfm-teams');
+  }
+
+  /**
+   * Finds a tfm team by the team id
+   * @param teamId - The team id to search for
+   * @returns The found team or null if not found
+   */
+  public static async findOneByTeamId(teamId: string): Promise<TfmTeam | null> {
```

Comment:
Because we don't perform any validation at the controller level to ensure this value is a `TeamId`, and the controller handles the case where we can't find a team which wouldn't make sense if we did validate the `teamId` (it would be a `500` instead of a `404`)